tox.ini

```diff
@@ -20,6 +22,7 @@ setenv =
 deps =
         django22: Django>=2.2,<3.0
         django30: Django>=3.0,<3.1
+        django31: Django>=3.0a1,<3.2
```

Comment:
I guess this should be `Django>=3.1a1,<3.2`, right?

Reply:
damn yup!